src/api/python/cvc5.pxi

```diff
@@ -501,19 +507,28 @@ cdef class Solver:
         sort.csort = self.csolver.mkDatatypeSort(dtypedecl.cdd)
         return sort
 
-    def mkDatatypeSorts(self, list dtypedecls, unresolvedSorts):
-        sorts = []
+    def mkDatatypeSorts(self, list dtypedecls, unresolvedSorts = None):
+        '''
+        Supports the following arguments:
+        std::vector<Sort> mkDatatypeSorts(const std::vector<DatatypeDecl>& dtypedecls)
+        std::vector<Sort> mkDatatypeSorts(const std::vector<DatatypeDecl>& dtypedecls, const std::c_set<Sort>& unresolvedSorts)
+        '''
```

Comment:
```suggestion
```

I think that we will need a different kind of comment here. Something similar to https://github.com/cvc5/cvc5/blob/320c48b1c82feb7a5d55af39d0cdbb7a1c2c6ff2/src/api/python/cvc5.pxi#L127 (can be done in a future PR).

Reply:
done (please see new comment)